qdrant-landing/content/documentation/guides/distributed_deployment.md

```diff
@@ -11,6 +11,22 @@ aliases:
 Since version v0.8.0 Qdrant supports a distributed deployment mode.
 In this mode, multiple Qdrant services communicate with each other to distribute the data across the peers to extend the storage capabilities and increase stability.
 
+## How many Qdrant nodes should I run?
```

Comment:
Would it make sense to organize this in a table?

Alternatively, if a table is too complicated, maybe set up bullets, with highlighted values, such as:

```
The number of Qdrant nodes you want depends on your priorities:

- **Costs**: If costs are most important, run a single Qdrant node. Drawbacks:
  - Downtime: Users will experience downtime during node restarts.
  - Performance: Limited to the resources of a single server.
  - Resilience: Recovery is not possible unless you have backups or snapshots.

- **Less downtime**: Run two Qdrant nodes with replicated shards. Drawbacks:
   - The cluster still responds to most read/write requests even when one node fails.
   - The cluster cannot run operations on collections.
   - Increased performance/scalability. You need not replicate shards to every node.
   - Increased performance with less RAM or disk space. However, this comes at a cost: resilience suffers, since non-replicated shards cannot be served when the node holding that shard is down.
   - Since collection operations are rarely needed, two-node clusters are a good balance between resilience and cost.

- **Best performance/scalability**: Three or more Qdrant nodes.
   - Assumption: Shards are replicated to at least two nodes. 
   - All operations continue to work when one node is down
   - Increased performance/scalability from multiple nodes.
   
```